Automattic/jetpack
# Add inline module settings for plan welcome page

Fixes #5909

#### Changes proposed in this Pull Request:

* Show toggles to enable plan features in the "welcome" page for the plan, after upgrade (as per #8700)

#### Testing instructions:

* Upgrade to Jetpack Personal
* Observe that form includes module toggles (toggle all on, then all off)
* Upgrade to Jetpack Premium
* Same as above
* Upgrade to Jetpack Professional
* Same as above

#### Screenshots

I have deliberately done the smallest change I can, while still sharing code across plans (which changes some copy). I have only removed prompts from the bottom of the form where I feel that there was a clearly redundant CTA.

### Personal

![goldsounds ngrok io_wp-admin_admin php_page jetpack ipad pro 3](https://user-images.githubusercontent.com/51896/37115465-0e3ac8c4-2200-11e8-9bce-134760d203f1.png)

### Premium

![goldsounds ngrok io_wp-admin_admin php_page jetpack ipad pro 2](https://user-images.githubusercontent.com/51896/37115477-158af464-2200-11e8-94dd-c486e16184f7.png)

### Professional

![goldsounds ngrok io_wp-admin_admin php_page jetpack ipad pro 1](https://user-images.githubusercontent.com/51896/37115504-21fe8378-2200-11e8-8369-0b6cca4b2df9.png)
